client/homebrew/pages/basePages/listPage/brewItem/brewItem.less

```
@@ -19,6 +19,20 @@
 	break-inside                : avoid;
 	box-shadow                  : 0px 4px 5px 0px #333;
 	background-color            : #cab2802e;
+	.thumbnail  {
+		width: 80px;
+		height: 100px;
+		border: 2px solid #58180D;
+		border-radius: 8px;
+		float: left;
+		margin-right: 5px;
+		display: flex;
+		img.thumbnailImage {
+			max-width: 100%;
+			max-height: 100%;
+			margin: auto;
```

Similar behavior but grows the image to fill the whole box and crops the excess instead of shrinking to fit. Might look a little cleaner instead of the empty space inside the box.

```suggestion
		img.thumbnailImage {
			width : 100%;
			height : 100%;
			object-fit : cover;
			border-radius : inherit;
```